src/google/protobuf/extension_set.h

```diff
@@ -664,7 +664,11 @@ class PROTOBUF_EXPORT ExtensionSet {
   // Grows the flat_capacity_.
   // If flat_capacity_ > kMaximumFlatCapacity, converts to LargeMap.
   void GrowCapacity(size_t minimum_new_capacity);
+  #ifndef __CUDACC__
```

Comment:
I think this condition is backwards, shouldn't it be ifdef instead of ifndef?

Reply:
yep.